src/index.js

```diff
@@ -158,21 +158,9 @@ function jumpToPast (history, index) {
 }
 // /jumpToPast
 
-// wrapState: for backwards compatibility to 0.4
-function wrapState (state) {
-  return {
-    ...state,
-    history: state
-  }
-}
-// /wrapState
-
 // updateState
 function updateState (state, history) {
```

Comment:
Alright, sounds good. :grin: If you change this, please make sure it doesn't break anything. :sweat_smile: